computer_vision/flip_augmentation.py

```diff
@@ -0,0 +1,134 @@
+import glob
+import os
+import random
+from string import ascii_lowercase, digits
+
+import cv2
+
+
+"""
+Flip image and bounding box for computer vision task
+https://paperswithcode.com/method/randomhorizontalflip
+"""
+
+# Params
+LABEL_DIR = ""
+IMAGE_DIR = ""
+OUTPUT_DIR = ""
+FLIP_TYPE = 1  # (0 is vertical, 1 is horizontal)
+
+
+def main() -> None:
+    """
+    Get images list and annotations list from input dir.
+    Update new images and annotations.
+    Save images and annotations in output dir.
+    >>> pass  # A doctest is not possible for this function.
+    """
+    img_paths, annos = get_dataset(LABEL_DIR, IMAGE_DIR)
+    print("Processing...")
+    new_image, new_annos, path = update_image_and_anno(img_paths, annos, FLIP_TYPE)
```

Comment:
It is better to use the _plural form_ for lists of similar things so you can do `for path in paths:`
```suggestion
    new_images, new_annos, paths = update_image_and_anno(img_paths, annos, FLIP_TYPE)
```